config/settings.yml

```diff
@@ -797,8 +797,10 @@ vanotify:
         form526_confirmation_email: fake_template_id
         covid_vaccine_registration: fake_template_id
     lighthouse:
-      disconnection_template: fake_template_id
-      connection_template: fake_template_id
+      api_key: fake_secret
+      template_id:
```

Comment:
We discussed changing it to templates.
Is it in separate PR?

Reply:
No, We have to modify devops as well. I think just modifying the devops template_id to templates will break vets-api aocde unless we use a new feature flag.  I can do it, but it might mean two PRs on vets-api and one PR in devops.